vue-frontend/src/components/contact/ThankyouScheduleMeeting.vue

```diff
@@ -0,0 +1,178 @@
+        </li>
+        <li>
+          If you have any questions or concerns, please don't hesitate to reach
+          out to me directly. I'm here to help and support you throughout the
+          process.
+        </li>
+        <li>
+          Thank you again for considering our company for your needs. We look
+          forward to the opportunity to work with you and to build a long and
+          successful partnership.
+        </li>
+      </ul>
+
+      <div class="tw-flex tw-items-end tw-flex-col tw-mt-10">
+        <p
+          class="tw-text-left v2-canopas-gradient-text tw-font-inter-bold tw-text-[1.5rem] md:tw-text-[1.688rem] xl:tw-text-[1.875rem] tw-leading-[2.25rem] md:tw-leading-[2.5rem] xl:tw-leading-[2.813rem]"
+        >
+          Thanks, <br /><span
+            class="tw-mt-0 tw-font-inter-medium tw-text-[1.125rem] md:tw-text-[1.25rem] xl:tw-text-[1.5rem] tw-leading-[1.688rem] md:tw-leading-[2rem] xl:tw-leading-[2.25rem] tw-text-black-core/[0.87]"
+          >
+            Jimmy</span
+          >
+        </p>
+      </div>
+    </div>
+    <!-- Show Calendly Iframe -->
+    <div v-if="openCalendlyIframeModal">
+      <transition name="modal">
+        <div
+          class="modal-mask tw-fixed tw-top-0 tw-left-0 tw-w-full tw-h-full tw-table mask tw-z-[1] tw-bg-[#00000080] tw-z-[5]"
+        >
+          <div
+            class="tw-flex tw-mx-auto tw-left-auto sm:tw-mx-auto tw-h-full login-modal modal-xl"
+            role="document"
+          >
+            <div
+              class="tw-relative tw-flex tw-flex-col tw-w-full tw-border-1 tw-border-gray tw-border-solid tw-rounded-md tw-rounded-3xl tw-bg-white tw-bg-clip-padding tw-outline-0 tw-p-2.5"
+            >
+              <div class="tw-relative tw-flex-auto tw-p-4">
+                <CalendlyIframe class="tw-w-full tw-h-screen" />
+
+                <button
+                  type="button"
+                  class="tw-absolute tw-right-2.5 tw-top-0 close modal-close-btn tw-border-none tw-text-pink-300 tw-text-4xl tw-font-light tw-bg-transparent focus:tw-outline-none"
+                  data-dismiss="modal"
+                  aria-label="Close"
+                >
+                  <span aria-hidden="true" @click="closeCalendlyIframeModal"
+                    >&times;</span
+                  >
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </transition>
+    </div>
+  </section>
+</template>
+
+<script>
+import image400w from "@/assets/images/client-thankyou/CTA-image-400w.webp";
+import image800w from "@/assets/images/client-thankyou/CTA-image-800w.webp";
+import ctaimage400w from "@/assets/images/client-thankyou/cta-image-400w.webp";
+import ctaimage800w from "@/assets/images/client-thankyou/cta-image-800w.webp";
+import ctaimage1400w from "@/assets/images/client-thankyou/cta-image-1-400w.webp";
+import ctaimage1800w from "@/assets/images/client-thankyou/cta-image-1-800w.webp";
```

Comment:
We have only 2 images in this section—and their width may not be greater than 500px. What is the use of this all images?